tests/unit_tests/cli/test_integration_cli.py

```diff
@@ -409,5 +409,7 @@ def test_that_prior_is_not_overwritten_in_ensemble_experiment(
 
         FeatureToggling.update_from_args(parsed)
         run_cli(parsed)
-        parameter_values = facade.load_all_gen_kw_data("default")
+        post_ert = EnKFMain(ResConfig("poly.ert"))
+        post_facade = LibresFacade(post_ert)
+        parameter_values = post_facade.load_all_gen_kw_data("default")
```

Comment:
Could you explain the reasoning behind why it is necessary to create a new instance?